### PR TITLE
Remove data-slidable for tabs

### PR DESCRIPTION
**Pull request by w20k**
Fix for https://github.com/octobercms/october/issues/4037

### CODE REVIEW

**Comment by w20k** [comment]
@LukeTowers  should I add a note to the Docs?

**Comment by LukeTowers** [comment]
@w20k sure? I'm not super familiar with what's going on here.

**Comment by w20k** [comment]
@LukeTowers by default, tables were swipeable and scrollable. Now only scrollable.

**Comment by LukeTowers** [comment]
@w20k what's the status on this?

**Comment by w20k** [comment]
@LukeTowers this is done.

**Comment by LukeTowers** [comment]
I still don't understand what's going on here

**Comment by w20k** [comment]
Related to this comment https://github.com/octobercms/october/issues/4037#issuecomment-458900265

**Comment by LukeTowers** [comment]
Could you summarize the issue for me?

**Comment by w20k** [comment]
Sure, Luke ;)

**Comment by w20k** [comment]
@LukeTowers shortly - for mobile there are two options right now swipe and scroll. Can't have both. Scrolling is enough ;)

**Comment by LukeTowers** [comment]
What does the slidable behaviour do?

**Comment by w20k** [comment]
@LukeTowers allows the tabs to be switched with the swipe gesture on touch devices. But when tab has any overflowing content or responsive elements that allow horizontal scrolling JS triggers swipe and switches tabs.
